src/execution/collectFields.ts

```diff
@@ -66,12 +67,14 @@ interface CollectFieldsContext {
  *
  * @internal
  */
+// eslint-disable-next-line @typescript-eslint/max-params
 export function collectFields(
   schema: GraphQLSchema,
   fragments: ObjMap<FragmentDetails>,
   variableValues: VariableValues,
   runtimeType: GraphQLObjectType,
   operation: OperationDefinitionNode,
+  maskSuggestions: boolean,
```

Comment:
We could replace this with an options object, the coverage aspect will be a bit harder to achieve though

Reply:
i think this is fine for now